tests/cli/init.sh

```diff
@@ -4,103 +4,103 @@ CLIENT_EXE=basecli
 SERVER_EXE=basecoin
 
```

Comment:
Okay, please use 4 spaces as tabs on bash files you were using two. More in the coding guide: https://github.com/tendermint/coding/blob/master/Coding_Standard.md#non-golang-code
I have this automatically built in to format when I save a `.sh` in vim